cypress/e2e/OrangeHRM/TC-03_VerifyAddEmployeeAPIOnOrangeHRM/TC-03_VerifyAddEmployeeAPIOnOrangeHRM.ts

```diff
@@ -53,4 +53,19 @@ When("the user navigate to Employee List page", () => {
 
 Then("the emolyee should be added successfully", () => {
     addEmployeeAssertions.checkTheEmployeeRecord([employeeId,"Rahaf","Jumaa"],true);
+});
+
+after(() => {
+  addEmployeeAPI.searchOnEmployeeByID(employeeId).then((response)=>
+  {
+      if(response.data[0].employeeId === employeeId){
+    addEmployeeAPI.DeleteEmployee(
+      deleteEmployee = {
+      "ids" : [empNumber]
+  });
+  }
+     else {
+      return;
+     }  
```

Comment:
GetEmployeeByID or the search should be in the delete function to be used in everywhere you want 
I mean in the after hook just call the delete API and inside this delete API you get the employee by id 

and why you declare const addEmployeeAPI = **new api();** in this way?
 and the name **addEmployeeAPI** not clear enough to use it to call The delete method